syntax = "proto2";

message Response_PasswordSalt {
    extend Response {
        optional Response_PasswordSalt ext = 1017;
    }
    optional string password_salt = 1;
}
